client/my-sites/marketing/tools/google-adwords.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import page from 'page';
+import PropTypes from 'prop-types';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import Button from 'components/button';
+import { getSelectedSite } from 'state/ui/selectors';
+import GoogleVoucherDetails from 'my-sites/checkout/checkout-thank-you/google-voucher';
+import { isPremium, isBusiness, isEcommerce, isEnterprise } from 'lib/products-values';
+import isSiteAtomic from 'state/selectors/is-site-automated-transfer';
+import MarketingToolsFeature from './feature';
+import { PLAN_PREMIUM } from 'lib/plans/constants';
+import QuerySiteVouchers from 'components/data/query-site-vouchers';
+import { recordTracksEvent as recordTracksEventAction } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface ConnectedProps {
+	isAtomic: boolean;
+	isJetpack: boolean;
+	isPremiumOrHigher: boolean;
```

Comment:
not a thing I want to change now, but I'm recognizing that we'll be able to relax our naming conventions on things like booleans the more we type. right now we use the variable name to distinguish between "Jetpack the thing" and "the site is a Jetpack site" but a variable named `jetpack` of type `boolean` covers that same communication need for us and automatically so